build.gradle

```diff
@@ -24,6 +25,15 @@ nexusStaging {
     packageGroup = "eu.xenit"
 }
 
+apply plugin: "org.sonarqube"
+
+sonarqube {
+    properties {
+        property "sonar.projectName", project.name
+        property "sonar.projectKey", project.name
```

Comment:
Do not set these properties here, but put the projectKey in the travis command

Reply:
why? this information is project specific, makes sense to keep with the `build.gradle`, it's already defined there.
In Travis script on the other hand I want to keep the information where and how the build will be executed (be it sonarcloud.io or self-hosted).

Reply:
`projectKey` is the project ID for sonarcloud/sonarqube. It is like the username of an account (and `sonar.login` passed on the commandline is the password)

Next to the fact that is should be a static string, and not derived from `project.name`, it also has no business being in the buildscript itself, given that it must be changed if a fork is made and they want to publish to sonarcloud/qube.